subsys/usb/class/bluetooth.c

```diff
@@ -137,6 +145,34 @@ static struct usb_ep_cfg_data bluetooth_ep_data[] = {
 	},
 };
 
+#define H4_CMD 0x01
+#define H4_ACL 0x02
+#define H4_SCO 0x03
+#define H4_EVT 0x04
+
+static void bt_h4_send(struct net_buf *buf)
```

Comment:
It depends the way you look at it, I had the opposite meaning on purpose since bt_h4_send is internal to the usb driver the same way hci_rx_thread to receiving from the stack, but perhaps having it as usb_h4_send would have been clearer.

Reply:
Yeah, I think usb_h4_send/recv would be clearer.